tests/loaders/test_configloader.py

```diff
@@ -62,3 +62,95 @@ def test_config_loader_load_correct_yaml(self, configloader):
 
         for key in data.keys():
             assert data[key] == config[key]
+
+    def test_config_loader_load_mapping_efficientdet(self, configloader):
+        node_name = "model.efficientdet"
+        test_map = configloader._load_mapping(node_name)
+
+        assert test_map["person"] == 0
+        assert test_map["parking meter"] == 13
+        assert test_map["toothbrush"] == 89
+
+    def test_config_loader_load_mapping_yolo(self, configloader):
+        node_name = "model.yolo"
```

Comment:
Since this is going into node-v1.2 which already contains yolox, consider adding tests for mapping for yolox model. Perhaps a fixture like
```
@pytest.fixture(params=["model.yolo", "model.yolox"])
def yolo_node_name(request):
    return request.param
```
So we can just increment that for more models which uses the yolo numbering in the future instead having to add new test functions for each new model.

Applicable to the other yolo related mapping tests below.

Reply:
My working branch is not sync with node-v1.2, so doesn't have YoloX. Merging with node-v1.2 will introduce many changes.
Suggest to postpone writing the YoloX unit tests until this has been merged into node-v1.2 (keep this as open issue).

Reply:
If you meant introducing changes that will show as "Files Changed" in this PR, merging the latest node-v1.2 (target branch) should not do that. But sure, I think we can write the YOLOX tests after this is merged.